build.csx

```diff
@@ -57,6 +58,8 @@ var gitlink = "./packages/gitlink.2.3.0/lib/net45/GitLink.exe";
 var xunit = "./packages/xunit.runner.console.2.0.0/tools/xunit.console.exe";
 
 // artifact locations
+var coverityDirectory = "./artifacts/coverity";
+var coverityResultsDirectory = "./artifacts/coverity/cov-int";

```

Comment:
No idea. But it's [mandated](https://scan.coverity.com/download):

> **Note** - Do not change the name of 'cov-int' intermediate directory.

and similar notes elsewhere on the page